ui/raidboss/data/05-shb/raid/e1s.txt

```diff
@@ -46,16 +46,16 @@ hideall "--sync--"
 500.0 "Eternal Breath" sync /:Eden Prime:3D8C:/ window 500,0
 510.0 "--targetable--"
 518.4 "Paradise Regained" sync /:Eden Prime:44FC:/
-528.8 "Vice And Virtue! (T)" sync /:Eden Prime:3D78:/
-542.5 "Vice And Virtue! (D)" sync /:Eden Prime:3D7A:/
+528.8 "Vice And Virtue (T)" sync /:Eden Prime:3D78:/
```

Comment:
These all have exclamation points to differentiate the powered up version from the original ones.

Reply:
Oh you are right they even have different spell ids. I guess I just assumed wrong then